cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -29,6 +29,20 @@
 namespace gluten {
 
 namespace {
+#define LOG_VALIDATION_MSG_FROM_EXCEPTION(err)                                                                                       \
+  logValidateMsg(fmt::format(                                                                                                        \
+      "Validation failed due to exception caught at file:{} line:{} function:{}, thrown from file:{} line:{} function:{} reason:{}", \
```

Comment:
It may be better to contain exception position if debug is enabled? Ditto for LOG_VALIDATION_MSG.

Reply:
I prefer keeping these information to help us quickly find root fallback cause, we may not require customer re-run with debug mode enabled, customer's resources and patience not always enough.